lib/fog/core/errors.rb

```diff
@@ -11,8 +11,6 @@ def self.slurp(error, message = nil)
       end
     end
 
-    class MockNotImplemented < Fog::Errors::Error; end
```

Comment:
Yes I was thinking the same thing too. As you said it's used only by Mock, therefore it should be in its scope.